docs/tutorial/electron-versioning.md

```diff
@@ -77,13 +77,10 @@ Whatever you choose, you will periodically have to bump the version in your `pac
 
 The process is as follows:
 
-1. All new major and minor releases lines begin with a `-beta.N` tag for `N >= 1`. At that point, the feature set is **locked**. That release line admits no further features, and focuses only on security and stability.
-e.g. `2.0.0-beta.1`.
-2. Bug fixes, regression fixes, and security patches can be admitted. Upon doing so, a new beta is released incrementing `N`.
-e.g. `2.0.0-beta.2`
-3. If a particular beta release is _generally regarded_ as stable, it will be re-released as a stable build, changing only the version information.
-e.g. `2.0.0`.
-4. If future bug fixes or security patches need to be made once a release is stable, they are applied and the _patch_ version is incremented accordingly
+1. All new major and minor releases lines begin with a beta series indicated by semver prerelease tags of `beta.N`, e.g. `2.0.0-beta.1`. Only backwards-compatible changes are admitted after the first beta.
```

Comment:
Do we really need beta cycles for minor releases?
It looks like it's too complicated to have two betas at the same time.

Allowing only "safe" features to minor releases can be a good alternative.

Reply:
> Do we really need beta cycles for minor releases?

This PR doesn't change existing policy on betas in minors.

My main answer to this question is I don't think it matters because we we won't be shipping supported minor releases for the forseeable future -- we have to catch the majors up with upstream first.

My default answer is 'yes we should have betas' because it's a risky proposition to ship code that hasn't been user-tested. We should revisit this question if/when it becomes a practical issue, since maybe the nightly builds will mitigate some of that risk.

> Allowing only "safe" features to minor releases can be a good alternative.

As voted by consensus in the maintainers group meetings?

Reply:
> This PR doesn't change existing policy on betas in minors.

But it _is_ policy related.
And I think it would be great to change policies regarding minor releases too.

> it's risky to ship code that hasn't been user-tested

I don't see any difference between non user-tested bugfixes and non user-tested features.
A change type should only dictate a next version number, not to be a criterion (TIL a new word!) for backporting.